test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -615,7 +616,8 @@ protected bool CheckEqualityFromPathsCore(string relPath, string basePath, strin
 
         private static void GetNumbersFromFile(ref string firstString, ref string secondString, decimal precision)
         {
-            Regex _matchNumer = new Regex(@"\b[0-9]+\.?[0-9]*\b", RegexOptions.IgnoreCase | RegexOptions.Compiled);
+            string decimalSeparator = CultureInfo.CurrentCulture.NumberFormat.NumberDecimalSeparator;
+            Regex _matchNumer = new Regex(@"\b[0-9]+\" + decimalSeparator + "?[0-9]*\b", RegexOptions.IgnoreCase | RegexOptions.Compiled);
```

Comment:
Don't you need to check for both characters?  The `secondString` will have the current culture's separator in it.  But the `firstString` will come from the baseline file, which probably has the InvariantCulture's separator in it (i.e. `.`).